### DISCUSSION TITLE
title bar

### DISCUSSION THREAD

**Discussion by oxy1000** [ocornut/imgui#4865] [Old/Archived discussions]
It works as intended but, is it the proper way of doing it? 🙄
![imgui](https://user-images.githubusercontent.com/94122366/148207913-2e732f5f-89f9-44dd-84b4-9e1286dc52be.png)
![titlebar](https://user-images.githubusercontent.com/94122366/148208093-ab7e96c7-0a32-48f6-b779-739af7759af5.png)

**Comment by rokups** [comment]
There is no proper way to customize a titlebar. You may get away by rendering manually into a titlebar rect using drawlist directly, but that will be prone to breaking in unexpected ways and generally unsupported. If you decide to go down that path, you may access window titlebar rect by calling `window->TitleBarRect()`.
